product_form_sale_link/views/sale_order_line.xml

```diff
@@ -6,16 +6,12 @@
         <field name="name">Sale Order Lines</field>
         <field name="res_model">sale.order.line</field>
         <field name="context">{}</field>
-        <field
-            name="domain"
-        >[('state', 'in', ['sale', 'done']), ('product_id.product_tmpl_id', 'in', active_ids)]</field>
+        <field name="domain">[('product_id.product_tmpl_id', 'in', active_ids)]</field>
     </record>
     <record id="action_product_product_sale_list" model="ir.actions.act_window">
         <field name="name">Sale Order Lines</field>
         <field name="res_model">sale.order.line</field>
         <field name="context">{}</field>
-        <field
-            name="domain"
-        >[('state', 'in', ['sale', 'done']), ('product_id', 'in', active_ids)]</field>
+        <field name="domain">[('product_id', 'in', active_ids)]</field>
```

Comment:
Then, I propose to apply a default filter for status sale

Reply:
Good point, I have added a default filter for it, thanks! :)